hibernate-types-52/src/test/java/com/vladmihalcea/hibernate/type/json/MySQLJsonTypeTest.java

```diff
@@ -53,6 +56,7 @@ public void test() {
             Event event = new Event();
             event.setId(1L);
             event.setLocation(location);
+            event.setAlternativeLocations(Arrays.asList(location));
```

Comment:
I think it's better to have a separate Test class for this new feature.

Reply:
Ok